client/my-sites/plugins/index.js

```diff
@@ -106,20 +108,33 @@ export default function () {
 		);
 
 		if ( config.isEnabled( 'marketplace-yoast' ) ) {
-			page( '/plugins/marketplace/domain/:site', renderDomainsPage, makeLayout, clientRender );
 			page(
-				'/plugins/marketplace/product/setup/:site',
+				'/marketplace/test/:site',
+				siteSelection,
+				navigation,
+				renderMarketplaceTestPage,
+				makeLayout,
+				clientRender
+			);
+
+			page( '/marketplace/domain/:site', renderDomainsPage, makeLayout, clientRender );
```

Comment:
To make those work, you'll need to add `/marketplace` here: https://github.com/Automattic/wp-calypso/blob/trunk/client/sections.js#L107

At this point, I'd totally recommend creating a new marketplace section, unless we really need to intermingle the plugins and the marketplace code.